retrofit-adapters/scala/pom.xml

```diff
@@ -69,6 +69,17 @@
           </signature>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
+        <configuration>
+          <archive>
+            <manifestEntries>
+              <Automatic-Module-Name>retrofit2.converter.scala</Automatic-Module-Name>
```

Comment:
I’m continuing a problem: the Scala adapter uses this package: `retrofit2.converter.scala`.

Reply:
Worth the grind to move it to the right package in follow up?

Reply:
It's unreleased, so yes.

Reply:
https://github.com/square/retrofit/pull/2639